tests/integrations/test_crossed_integrations.py

```diff
@@ -50,6 +47,19 @@ def get_span(interface, span_kind, topic):
         logger.debug("No span found")
         return None
 
+    @staticmethod
+    def get_topic(span) -> str | None:
+        """ Extracts the topic from a span by trying various fields """
+        topic = span["meta"].get("kafka.topic")  # this is in python
+        if topic is None:
+            if "Topic" in span["resource"]:
+                # in go and java, the topic is the last "word" of the resource name
+                topic = span["resource"].split(" ")[-1]
```

Comment:
In python 3 you can extract the last word of the string:

`_, *_, topic = span["resource"].split()`